js/widgets/fixedToolbar.js

```diff
@@ -54,17 +53,23 @@ define( [ "jquery", "../widget", "../core", "../animationComplete", "../navigati
 				this._makeFixed();
 			}
 			if ( this.options.position === "fixed" && !this.options.supportBlacklist() ) {
-				var $page = ( !!this.page )? this.page: ( $(".ui-page-active").length > 0 )? $(".ui-page-active"): $(".ui-page").eq(0);
+				var currentPage = ( !!this.page ) ? 
+					this.page : 
+					( $(".ui-page-active").length > 0 ) ? 
+					$(".ui-page-active") : 
+					$(".ui-page").eq(0);
 
 				if ( o.fullscreen !== undefined) {
 					if ( o.fullscreen ) {
-						this.element.addClass( "ui-"+ this.role +"-fullscreen" );
-						$page.addClass( "ui-page-" + this.role + "-fullscreen" );
+						this._addClass( "ui-toolbar-" + this.role + "-fullscreen" );
+						this._addClass( currentPage, null, "ui-page-" + this.role + "-fullscreen" );
```

Comment:
If we update above to use `ui-toolbar-` class for this then this will be a structure class now